grpcurl/tests/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+TESTDIR="$(dirname "${0}")"
+PLANDIR="$(dirname "${TESTDIR}")"
+SKIPBUILD=${SKIPBUILD:-0}
+
+source "${PLANDIR}/plan.sh"
+
+if [ "${SKIPBUILD}" -eq 0 ]; then
+  set -e
+  pushd "${PLANDIR}" > /dev/null
+  build
+  popd > /dev/null
+  set +e
+fi
+
+source "${PLANDIR}/results/last_build.env"
+hab pkg install --binlink --force "${PLANDIR}/results/${pkg_artifact}"
```

Comment:
The standard for running tests is to pass this script a package ident e.g.

```
hab pkg build grpcurl
source results/last_build.env
hab studio run "./grpcurl/tests/test.sh $pkg_ident"
```

That avoids sourcing anything inside the test studio, and also lets us re-use the test script later in the pipeline (when we may have a pkg_artifact to test but do not have a `last_build.env` )

Reply:
When I say "standard", I mean that we're working through adding tests following [this guide](https://github.com/habitat-sh/core-plans/blob/master/docs/dev/policy_documents/standards-for-tests.md), and we're hoping to merge https://github.com/habitat-sh/core-plans/pull/2573 soon to have `tests/test.sh` be executed in CI

Reply:
Thanks! I wasn't aware of these efforts. I'll push an update tomorrow.

Reply:
✔️ Updated.

Btw, I'd take care of `core/dex` and `core/opa` when I next touch them -- or would you prefer to get this standardization over with fast?